Modules/Server/Sources/PocketCastsServer/Public/Cache/EpisodeInfoHandler.swift

```diff
@@ -0,0 +1,119 @@
+import Foundation
+
+/// Request information about an episode using the show notes endpoint
+public class EpisodeInfoHandler {
+    public static var shared = EpisodeInfoHandler()
```

Comment:
Since this is being only being used by the `CacheServerHandler` which is a singleton, do we need to also make this a singleton?

Reply:
Done in 2a293eab80f2c00f3b1cb82c9a39f18d15137019